src/scripts/browserDeviceProfile.js

```diff
@@ -918,22 +918,22 @@ export default function (options) {
     let vp9VideoRangeTypes = 'SDR';
     let av1VideoRangeTypes = 'SDR';
 
+    if (supportsDolbyVision(options)) {
+        hevcVideoRangeTypes += '|DOVI';
+    }
+
     if (supportsHdr10(options)) {
-        hevcVideoRangeTypes += '|HDR10';
+        hevcVideoRangeTypes += '|HDR10|DOVI_with_HDR10_Fallback';
```

Comment:
If we can just stream DoVi (8?) to an HDR10-only client and it will automatically fallback to HDR10 (client-side), then we don't need a special value.

Reply:
Correct. https://github.com/jellyfin/jellyfin/issues/10468#issuecomment-1778532412

Reply:
I added some more context regarding this addition on the above thread, [here](https://github.com/jellyfin/jellyfin/issues/10468#issuecomment-1780029285). I don't think DoVi profile 8 playback can be enabled on webOS without changes in the server (or ffprobe), so added my comments there.